### PR TITLE
Add CertificatesSignInfo to asset manifest

### PR DESCRIPTION
**Pull request by chcosta**
Adds CertificatesSignInfo to asset manifest.

Validated with https://dnceng.visualstudio.com/internal/_build/results?buildId=810113&view=results

This doesn't update the merged manifest, I need to look into that next.

### CODE REVIEW

File: src/Microsoft.DotNet.Build.Tasks.Feed/src/BuildManifestUtil.cs [hunk 1]
```diff
@@ -1,6 +1,7 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using Amazon.S3;
```
Comment (mmitche) [inline src/Microsoft.DotNet.Build.Tasks.Feed/src/BuildManifestUtil.cs:4]: Needed?

Comment (chcosta) [reply src/Microsoft.DotNet.Build.Tasks.Feed/src/BuildManifestUtil.cs:4]: wtf?

**Review by mmitche — APPROVED** [review]

**Comment by chcosta** [comment]
Waiting to merge until @jcagme has updated Maestro to handle the merged manifest

**Comment by chcosta** [comment]
Validated the merged manifest changes will work with https://dnceng.visualstudio.com/internal/_build/results?buildId=811497&view=artifacts&type=publishedArtifacts

**Pull request merged**